src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void draw();
+
```

Comment:
Extra empty line

Reply:
Maven allowes to remove only line after void draw(); and does not clean package if line befor public interface Drawable { is removed.